project/Dependencies.scala

```diff
@@ -46,6 +46,9 @@ object Dependencies {
   val sbtCompilerInterface = sbtOrg % "interface" % sbtMainVersion
   val sbtCompletion        = sbtOrg % "completion" % sbtMainVersion
 
+  val coreNextVersion      = "0.1.0-M1"
+  val coreNext             = Def.setting { Defaults.sbtPluginExtra(sbtOrg % "core-next" % coreNextVersion, sbtBinaryVersion.value, scalaBinaryVersion.value) }
```

Comment:
It's a library that uses sbt's code base, so I'm currently shipping it as a plugin.